### ISSUE TITLE
Try passing information to functions by using parameters

### ISSUE THREAD

**Issue by Razpudding** [JiDarwish/javascript1#12]
You can just pass the list of books to the function as a param like we taught you.

**Comment by JiDarwish** [comment]
Yes indeed I can but in this situation it seems to be working fine, I thought about passing info as parameters but then I thought it wasn't needed. Do you mean that it's bad to do this?
